### PR TITLE
monitoring

### PR DESCRIPTION
**Pull request by ashotland**

### CODE REVIEW

File: docs/cloud/monitoring.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+
+# Monitoring
+Dragonfly Cloud provides data stores observability through a readily available Grafana based [public dashboard](#public-dashboard), and a [Prometheus compatible metrics endpoint](#prometheus-compatible-metrics-endpoint) that you can scrape and integrate with most monitoring and alerts solutions like Datadog and Grafana. 
+
+If you didn't create a datastore yet, you see [Datastores](/docs/cloud/datastores) to create your first datastore.
+
+## Public Dashboard
+
+To open a datastore’s public dashboard click the dashboard icon [TODO ask for help adding the icon it is available here https://fonts.google.com/icons?selected=Material+Symbols+Outlined:monitoring:FILL@1;wght@400;GRAD@0;opsz@24&icon.query=monitoring&icon.size=24&icon.color=%23e8eaed ]
```
Comment (ashotland) [inline docs/cloud/monitoring.md:13]: Hi @Niennienzz - maybe you can help me with adding this icon inline here?

Comment (ashotland) [reply docs/cloud/monitoring.md:13]: @AnzeKop as Joe is out maybe you can help me here?

Comment (AnzeKop) [reply docs/cloud/monitoring.md:13]: Hey @ashotland yea definitely can help. Just want to add the icon inline into the text?

Comment (AnzeKop) [reply docs/cloud/monitoring.md:13]: Something like this?
Screenshot 2024-06-26 at 09 23 39

Comment (ashotland) [reply docs/cloud/monitoring.md]: Thanks @AnzeKop !

Comment (ashotland) [reply docs/cloud/monitoring.md]: Hi @AnzeKop - same for me builds locally but get [ERROR] Error: Docusaurus found broken links!
 on Vercel 
 
I've attached the svg file to this comment - can you please show me how to add it a as  direct link ?

monitoring_24dp_FILL1_wght400_GRAD0_opsz24

Comment (AnzeKop) [reply docs/cloud/monitoring.md]: @ashotland you can write straight jsx into the .md files as docasurus preprocesses it.

For the link you've shared above I just copied in the raw svg code but if you have a live url to the image you could also just use the image tag. Just make sure to use jsx syntax for any inline styles. If you want to wrap it an a clickable link you can wrap an anchor tag around it.

```jsx
<a href="">
<img src="" alt="" />
</a>
```

Comment (AnzeKop) [reply docs/cloud/monitoring.md]: Also try this for the link
```md
[Datastores](./datastores)
```

Comment (ashotland) [reply docs/cloud/monitoring.md]: Thanks that fixed the build :)

**Pull request merged**